cypress/integration/tests/ui-tests/ikeaMobileProduct.spec.ts

```diff
@@ -2,6 +2,9 @@ import { merchantProductPage } from 'cypress/page-objects/merchantProductPage'
 
 describe('Ikea Mobile Product Page Injection Test', () => {
   beforeEach(() => {
+    Cypress.on('uncaught:exception', (err, runnable) => {
+      return false
+    })
```

Comment:
Because we are injecting the widget into a merchant website (with its own exceptions), this line will stabilize the tests.